lib/CleantalkSP/SpbctWP/RemoteCalls.php

```diff
@@ -489,4 +489,39 @@ public static function action__private_record_delete() // phpcs:ignore PSR1.Meth
     {
         return spbct_sfw_private_records_handler('delete');
     }
+
+    /**
+     * Handle remote call action "run_service_template_get".
+     * @return string
+     * @throws \InvalidArgumentException
+     * @psalm-suppress PossiblyUnusedMethod
+     */
+    public static function action__run_service_template_get() // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    {
+        global $apbct;
```

Comment:
This is another plugin State class instance.

Reply:
fixed